.github/workflows/job_build_linux.yml

```diff
@@ -178,31 +178,23 @@ jobs:
           cmake -DCMAKE_INSTALL_PREFIX=${INSTALL_DIR} -P ${BUILD_DIR}/cmake_install.cmake
           cmake -DCMAKE_INSTALL_PREFIX=${INSTALL_TEST_DIR} -DCOMPONENT=tests -P ${BUILD_DIR}/cmake_install.cmake
           cmake -DCMAKE_INSTALL_PREFIX=${DEVELOPER_PACKAGE_DIR} -DCOMPONENT=developer_package -P ${BUILD_DIR}/cmake_install.cmake
-
-      - name: Pack Artifacts
-        run: |
-
+          
           # Add the ONNX Runtime version and skip tests list to the archive to use in the ONNX Runtime Job
           # w/o the need to checkout repository
-
           cp -R ${ONNX_RUNTIME_UTILS} ${INSTALL_DIR}
-
-          pushd ${INSTALL_DIR}
-            tar -I pigz -cvf ${BUILD_DIR}/openvino_package.tar.gz *
-          popd
-
-          pushd ${DEVELOPER_PACKAGE_DIR}
-            tar -I pigz -cvf ${BUILD_DIR}/openvino_developer_package.tar.gz *
-          popd
           
-          pushd ${BUILD_DIR}/wheels
-            tar -I pigz -cvf ${BUILD_DIR}/openvino_wheels.tar.gz *
-          popd
-
-          pushd ${INSTALL_TEST_DIR}
-            tar -I pigz -cvf ${BUILD_DIR}/openvino_tests.tar.gz *
-          popd
-
+      - name: Pack openvino_package
+        run: tar -I pigz -cvf ${BUILD_DIR}/openvino_package.tar.gz *
+        working-directory: ${{ env.INSTALL_DIR }}
+        
+      - name: Pack openvino_developer_package
+        run: tar -I pigz -cvf ${BUILD_DIR}/openvino_developer_package.tar.gz *
+        working-directory: ${{ env.DEVELOPER_PACKAGE_DIR }}
+        
+      - name: Pack openvino_tests
+        run: tar -I pigz -cvf ${BUILD_DIR}/openvino_tests.tar.gz *
+        working-directory: ${{ env.INSTALL_TEST_DIR }}
```

Comment:
should we pack wheels as dedicated package? they are typically not a part of OpenVINO release package

Reply:
yes, we should, I'm working on that now, the last changes did introduce that, but still need to modify pipelines

Reply:
done